src/Runner.Worker/Handlers/OutputManager.cs

```diff
@@ -139,12 +139,10 @@ public void OnDataReceived(object sender, ProcessDataReceivedEventArgs e)
 
                         // Convert to issue
                         var issue = ConvertToIssue(match);
-
                         if (issue != null)
                         {
                             // Log issue
-                            _executionContext.AddIssue(issue, stripped);
```

Comment:
This was the only place we were passing a message override to `AddIssue`

What's interesting is that, as far as I can tell, it was unnecessary.

Have a look here.